app/javascript/components/components/Header.js

```diff
@@ -2,40 +2,49 @@ import React, { Component } from 'react'
 import { NavLink } from "react-router-dom"
 import { Nav, NavItem } from "reactstrap"
 class Header extends Component {
-  render(){
+  render() {
     const {
       logged_in,
```

Comment:
The javascript convention is to use pascal case `loggedIn` instead of the snake case `logged_in` you can convert to the convention here by doing `logged_in: loggedIn` this is called aliasing and is a feature of destructuring.

Reply:
It also looks like you're not using all these... no need to destructure the ones you don't need.

Reply:
I updated the syntax and deleted what we weren't using.